presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -290,7 +291,8 @@ public ConnectorPageSource getPageSource(
                     ORC,
                     orcFileTailSource,
                     stripeMetadataSource,
-                    new OrcReaderOptions(readerAttributes.getMaxMergeDistance(), readerAttributes.getTinyStripeThreshold(), HUGE_MAX_READ_BLOCK_SIZE, readerAttributes.isZstdJniDecompressionEnabled()));
+                    new OrcReaderOptions(readerAttributes.getMaxMergeDistance(), readerAttributes.getTinyStripeThreshold(), HUGE_MAX_READ_BLOCK_SIZE, readerAttributes.isZstdJniDecompressionEnabled()),
+                    new HiveFileContext(false, Optional.empty()));
```

Comment:
For Raptor we need to decide if to cache or not as well. Whether to cache or not should be determined by `RaptorPageSourceProvider::createPageSource` (where `SplitContext` will be passed in). Depends on which patch lands first. But having it default to false here is not right.